src/VisualStudio/VisualBasic/Impl/LanguageService/VisualBasicPackage.vb

```diff
@@ -67,7 +67,6 @@ Namespace Microsoft.VisualStudio.LanguageServices.VisualBasic
         Protected Overrides Async Function InitializeAsync(cancellationToken As CancellationToken, progress As IProgress(Of ServiceProgressData)) As Task
             Try
                 Await MyBase.InitializeAsync(cancellationToken, progress).ConfigureAwait(True)
-                Await JoinableTaskFactory.SwitchToMainThreadAsync(cancellationToken)
```

Comment:
>SwitchToMainThreadAsync [](http://example.com/codeflow?start=42&length=23)

Neither RegisterLanguageService nor RegisterService require the main thread.

Reply:
do we need/want CA(true) on the prior line?  if so, doc.